### ISSUE TITLE
OCRmyPDF broken with ghostscript 9.51

### ISSUE THREAD

**Issue by spwhitton** [jbarlow83/OCRmyPDF#508]
ghostscript 9.51 has been uploaded to Debian unstable and OCRmyPDF's test suite is failing on our CI infrastructure: https://ci.debian.net/packages/o/ocrmypdf/unstable/amd64/ (currently 9.6.0 is being tested but 9.6.1 seems to have the same problems).

Thanks!

**Comment by jbarlow83** [comment]
Looks like a Ghostscript regression.

**Comment by jbarlow83** [comment]
Probably two issues, this is one...
https://bugs.ghostscript.com/show_bug.cgi?id=702221

**Comment by jbarlow83** [comment]
Using the debian:sid Docker image, I get

```
root@d724bca35422:/# gs -dQUIET -dSAFER -dBATCH -dNOPAUSE -sDEVICE=txtwrite -dTextFormat=0 -dFirstPage=1 -dLastPage=1 -o - -f /local/tests/resources/skew.pdf   
jbig2dec FATAL ERROR decoding image: incompatible jbig2dec header (0.17) and library (0.18) versions 
   **** Error reading a content stream. The page may be incomplete.
               Output may be incorrect.
   **** Error: File did not complete the page properly and may be damaged.
               Output may be incorrect.
<page>
</page>
```

That suggests to me that something is wrong with how ghostscript 9.51-dfsg-1 is being built.

I tried using the binary release of gs 9.51 from Artifex (https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs951/ghostscript-9.51-linux-x86_64.tgz), and none of those errors appear.

I have not been able to reproduce the malloc issue (`malloc(): invalid size (unsorted)`) that appears in the logs. I came across this jbig2dec issue while trying to reproduce that one.

My recommendation would be for Debian to roll back to 9.50 because there seems to be multiple issues. Can I leave it to you to report this to Debian?

**Comment by jbarlow83** [comment]
Ghostscript 9.51 is now blacklisted from ocrmypdf due to this issue and other major problems in that release. See commit 00498282.

**Comment by jbarlow83** [comment]
Fixed with v9.7.1 and pikepdf v1.11.0
